client/src/components/content/CharacterShameTile.js

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import { Link } from "react-router-dom"
+
+const CharacterShameTile = props => {
+  const { id } = props.character
+
+  return (
+    <div className="shameTile centered">
+      <h2>SHAME ZONE</h2>
+      <h4>#1 Loser</h4>
+      <Link to={`/characters/${id}`}>
+        <p>{props.character.name} - Total votes: {props.character.sum}</p>
```

Comment:
I would recommend staying consistent with your destructuring. If we are already destructuring `id`, let's also destructure `name` and `sum`